app/controllers/api/v1/neighbour_responses_controller.rb

```diff
@@ -9,6 +9,10 @@ class NeighbourResponsesController < Api::V1::ApplicationController
       skip_before_action :authenticate_api_user!
 
       def create
+        unless @planning_application.application_type.consultation_steps.include? "neighbour"
```

Comment:
how about moving it on its own method?

Reply:
I'm not sure that there would be much benefit, when it's quite short and only used once.